model_templates/inference/python3_unstructured/custom.py

```diff
@@ -3,6 +3,12 @@ def load_model(input_dir):
 
 
 def score_unstructured(model, data, query, **kwargs):
+    print("Model: ", model)
+    print("Incoming content type params: ", kwargs)
+    print("Incoming data type: ", type(data))
+    print("Incoming data: ", data)
+
+    print("Incoming query params: ", query)
```

Comment:
Maybe print the query params before the data.. data might be long .. easier to see all args together.

Reply:
This is only test of the sake of examples.
They are empty in example1 and 2. I use them only in example 3.
That's why i put them in the end